### ISSUE TITLE
Message Box Highlight & Dimming

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1902]
Hi everyone! 

I noticed we show the input boxes for chat channels in bright white, that dim down upon hovering and typing: 

![Message Box 1](https://user-images.githubusercontent.com/85491794/192905327-7dc66029-4ea1-42d0-a0a2-6d1a5bafe1ed.png)

![Message Box 2](https://user-images.githubusercontent.com/85491794/192905339-013f033d-8bf1-415d-b5d0-4b6068608c66.png)

I suggest we do the contrary: we show a dimmed version that lights up upon hovering and typing, to follow the attention of the user.

**Comment by IanMayo** [comment]
Yes @ShendMikullovci - I spotted this too, yesterday.  Good suggestion.
